### DISCUSSION TITLE
Is it possible to exclude a mountpoint from the mountpoint sensor?

### DISCUSSION THREAD

**Discussion by dflvunoooooo** [joshuar/go-hass-agent#131] [Q&A] [answered]
I would like to exclude a certain path from the mountpoint sensor. I have snap installed and HA go-hass-agent recognizes around 20 mountpoints from snap. But they are always 100% filled and do not really need to be monitored.

**Comment by joshuar** [comment, marked as answer]
Not at the moment unfortunately. What you can do however is disable the entity/sensor in Home Assistant for that mountpoint which will hide it from being displayed and also stop the agent from sending updates for it.

To do that, you can follow the instructions for [customising entities](https://www.home-assistant.io/docs/configuration/customizing-devices/):

- Select the entity, either from the frontend or by selecting the info button next to the entity in the Developer Tools “States” tab.
- Select the cog icon in the right corner of the entity’s dialog.
- Toggle the "Enabled" switch.

The entity/sensor should then be hidden in Home Assistant. Next time the agent goes to send an update for that sensor, Home Assistant will tell it it is disabled and it will stop sending the data.

**Reply by dflvunoooooo** [reply]
Thank you for your answer. I see. I disabled them all by hand, thank you. A good to know, that go-hass-agent will stop sending data.
